user-guide/Advanced_Modules/Dashboards_and_Low_Code_Apps/GQI/Extensions/GQI_Custom_Operator.md

```diff
@@ -0,0 +1,12 @@
+---
+uid: GQI_Custom_Operator
+---
+
+# Configuring a custom operator for a query
+
+> [!WARNING]
+> This feature is currently still in preview. For more information, see [Soft-launch options](xref:SoftLaunchOptions).
+
+The custom operator is available in soft launch from DataMiner 10.x.x onwards, if the soft-launch option *GenericInterface* is enabled.
```

Comment:
The GQI Custom Operator was first introduced with [RN32878](https://intranet.skyline.be/DataMiner/Lists/Release%20Notes/DispForm2.aspx?ID=32878) so that would be feature release version 10.2.7.